roles/network_plugin/calico/tasks/upgrade.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: "Download calico-upgrade tool (force version)"
```

Comment:
This belongs in download role with type file. Look at kubeadm as an example.

Reply:
ok, will move this to download role

Reply:
actually we just want to download it when it detects it needs to upgrade calico, so we don't want to download it when no upgrade needed, and detection happens in calico role. so i think it should not be moved to download :/ or detection then should be moved somewhere else